### DISCUSSION TITLE
Copilot Community Check-in

### DISCUSSION THREAD

**Discussion by Akash1134** [community/community#48964] [Copilot News and Announcements]
Hey everyone!, Welcome to another Copilot check-in and I hope you’ve had a lovely ❤️ February. Are you ready to March forward with some exciting Copilot updates? Let’s get started by ✨-ing some of our amazing community members who stood out this past month.

## 💌  From the Community - Shoutout 📢  to these members

- A big round of applause 👏  for [@juanescacha](https://github.com/Juanescacha) for helping [@pyronaur](https://github.com/pyronaur) as their suggestions lead him to the solution to their [problem](https://github.com/orgs/community/discussions/47070) with Copilot. 
- [@tomivd](https://github.com/tomivd) reported an [issue](https://github.com/orgs/community/discussions/45685) with Copilot while using Codespaces. Our champion ⭐ [@jge162](https://github.com/jge162) nudged them in the right direction.
- We’re are on spree with self-solvers in the community. It’s just delightful to see so many members coming back to their posts and sharing a solution that can potentially help so many other users.
   - Thanks [@rafek1241](https://github.com/rafek1241) for sharing a [solution](https://github.com/orgs/community/discussions/47524) from VSCodium repo that worked for them to authorize Copilot for Business license. 🎉
   - Much deserving 👏 for [@onticinc](https://github.com/onticinc) for their dedication to working with different members and finally getting to the [root cause](https://github.com/orgs/community/discussions/46567#discussioncomment-5016788) of the issue. What an incredible self-solve 💯
   - ✨ for [@Xteuk](https://github.com/Xteuk) for doing a solid diagnosis of the issue and then keeping the community [updated](https://github.com/orgs/community/discussions/47638) with the solution.

## [Copilot for Business is generally available](https://github.blog/2023-02-14-github-copilot-for-business-is-now-available/) 👩‍✈️

It's been almost 2 weeks since we [announced](https://github.com/orgs/community/discussions/48284) the general availability of GitHub Copilot for Business. 

GitHub Copilot for Business is available to Free, Team, and GitHub Enterprise Cloud customers. This allows your organization developers access to GitHub Copilot’s powerful AI while providing administrators with license management and centralized policy controls on top of industry-leading privacy.

Some of the feature highlights are:

- [Improved Security Vulnerability Filter](https://github.blog/2023-02-14-github-copilot-now-has-a-better-ai-model-and-new-capabilities/)
- Ability to run Copilot and [Visual Studio IntelliCode](https://visualstudio.microsoft.com/services/intellicode/) side by side
- VPN Proxy Support
- Simple SaaS Sign-Ups

If you haven’t tried it yet, here’s how you can [enable GitHub Copilot](https://docs.github.com/en/enterprise-cloud@latest/copilot/overview-of-github-copilot/about-github-copilot-for-business#setting-up-github-copilot-for-business) for your business. 🚀

## Good Reads - Your dose 💊  of AI updates

- Dive into the powerful new improvements that we’ve made in GitHub Copilot 🚢  in [this blog post](https://github.blog/2023-02-14-github-copilot-now-has-a-better-ai-model-and-new-capabilities/) by [@shuyinzjk](https://github.blog/author/shuyinzjk/) where he has shared why you should start building with Copilot. 
- Supercharge ⚡ your Copilot learning journey and know the inside-out of your favorite pair programmer. [blackgirlbytes](https://github.com/blackgirlbytes) has put together a simple but descriptive [Introduction to Copilot](https://github.com/orgs/community/discussions/39086) that’s a readers delight.
- Wanna know more about Copilot and Copilot Labs? This post from [@mishmanners](https://github.com/mishmanners) is your one-stop shop 🛒  for [understanding your Code better with GitHub Copilot](https://dev.to/github/understand-your-code-using-github-copilot-5375). 

## Resources to 🔖 

- Copilot [Troubleshooting Resources](https://docs.github.com/en/copilot/troubleshooting-github-copilot) ⚙️ 

- [Getting Started](https://docs.github.com/en/copilot) with GitHub Copilot  🚥 

- [Features](https://github.com/features/copilot) and [FAQs](https://github.com/features/copilot#:~:text=Frequently%20asked%C2%A0questions) ❔ 

- [Blogpost](https://github.blog/) & [Project ReadMe](https://github.com/readme) 📖 

- [GitHub Public Roadmap](https://github.com/orgs/github/projects/4247/views/1) 🗺️

**Comment by rileyblackwell** [comment]
Great product, but adding a keyboard shortcut to toggle copilot on/off would be a very nice feature. There are times when you want to temporarily ignore Copilot's suggestions because they aren't accurate/useful. It would be less distracting to simply toggle copilot off and then when you reach a line of code where copilot is useful again toggle back on.

**Reply by rileyblackwell** [reply]
I should be more specific- Toggle on/off that doesn't have a window pop asking if you want to disable globally or for a particular language. It would be ideal if the keyboard shortcut immediately disables Copilot in the current workspace.

**Reply by Akash1134** [reply]
Thank you for sharing your feedback @rileyblackwell I recommend that you share your feedback in a [new discussion post](https://github.com/orgs/community/discussions/new?category=copilot) under the `Product Feedback` topic area so that our engineers can look into these.

**Comment by willjhenry** [comment]
would be nice to see a copilot road map, and also more notes when the extension is updated

**Reply by Akash1134** [reply]
Awesome idea @willjhenry We currently have a [GitHub product roadmap](https://github.com/orgs/github/projects/4247/views/1) that you can filter to see upcoming ships related to the Copilot product. Regarding, the extension updates, we have an in-product notification setup to let users know whenever a new version is shipped. Your suggestion to have it included in the roadmap is great for major updates.  Thanks!

**Comment by TheGamer1002** [comment]
I'd really like to see Copilot suggestions in the Intellicode menu. It's a feature that's present in my Neovim setup, and I think it's quite nifty. It also allows me to select other suggestions without the `Escape, Ctrl+Space` shortcut.

**Reply by Akash1134** [reply]
The Copilot team is paying attention to the feedback, feature requests & bugs we are hearing from users in the Community Discussions so please continue to share your thoughts.

- If you believe this should be a feature request or enhancement please [add a discussion on the topic to the Copilot Category](https://github.com/orgs/community/discussions/new?category=copilot) and include as much detail as you can to help inform the product strategy.
- Tips for posting in discussions: 1. Make sure you select the product feedback topic category. 2. Add a clear title related to the request. 3. If its a feature request add the enhancement label

**Comment by adrianisk** [comment]
Hi Akash, I have a question (maybe feature request?) about Copilot - is there a way I can see individual metrics on how many lines of code Copilot is generated when I'm using it in VSCode? Ideally I'd like to be able to track which lines of code in a Github repo were authored by Copilot (or in a specific commit), but I'd even be happy with just how many lines of code it's generated in total over a period of time. Is any of that data collected and available for end users? Thanks!

**Reply by Akash1134** [reply]
Hi @adrianisk Thank you for your feedback and this is a good-to-have feature. I have shared this with the engineers. For more details, please read our [FAQs](https://github.com/features/copilot#:~:text=Frequently%20asked%C2%A0questions)

**Comment by setzerbomb** [comment]
If I'm using github copilot business, does the Code Snippets Data policy apply only to repos of my organization or all the repos that I'm using on my local machine?

**Reply by Akash1134** [reply]
Hi @setzerbomb Great Question! GitHub Copilot transmits snippets of your code from your IDE to GitHub to provide suggestions to you. Code snippets data is only transmitted in real-time to return Suggestions, and is discarded once a suggestion is returned. 

Copilot for Business does not retain any Code Snippets Data and it does not train on your private repos regardless of where it originates including public, private and non-GitHub repos.

**Comment by Ameausoone** [comment]
It would be nice to have a suggestion for commit message based on commit content.

**Reply by Akash1134** [reply]
The Copilot team is paying attention to the feedback, feature requests & bugs we are hearing from users in the Community Discussions so please continue to share your thoughts.

- If you believe this should be a feature request or enhancement please [add a discussion on the topic to the Copilot Category](https://github.com/orgs/community/discussions/new?category=copilot) and include as much detail as you can to help inform the product strategy.
- Tips for posting in discussions: 1. Make sure you select the product feedback topic category. 2. Add a clear title related to the request. 3. If its a feature request add the enhancement label
